docs/content/3.forms/11.Rating.md

```diff
@@ -0,0 +1,96 @@
+---
+description: Display a rating field.
+links:
+  - label: GitHub
+    icon: i-simple-icons-github
+    to: https://github.com/nuxtlabs/ui/blob/dev/src/runtime/components/forms/Rating.vue
+navigation:
+  badge: New
+---
+
+## Usage
+
+Use a `v-model` to make the Rating reactive.
+
+::component-example
+#default
+:rating-example
+
+#code
+```vue
+<script setup>
+const rate = ref(0)
+</script>
+
+<template>
+  <URating v-model="rate" />
+</template>
+```
+::
+
+### Style
+
+Use the `color` prop to change the style of the Rating.
+
+::component-card
+---
+props:
+  color: 'primary'
+---
+::
+
+### Sizes
+
+Use the `size` prop to change the size of the icons.
+
+::component-card
+---
+props:
+  size: lg
+---
+::
+
+### Icon
+
+Use any icon from [Iconify](https://icones.js.org) by setting the `icon` prop by using this pattern: `i-{collection_name}-{icon_name}` or change it globally in `ui.rating.default.icon`.
+
+::component-card
+---
+props:
+  icon: 'i-heroicons-heart-20-solid'
+excludedProps:
+  - icon
+---
+::
```

Comment:
I suggest adding a `modelValue` with some start to enable users to easily preview options such as `readOnly`, `icon`, and `color`.

```md
::component-card
---
baseProps:
  modelValue: 3
  color: 'red' 
props:
  icon: 'i-heroicons-heart-20-solid'
excludedProps:
  - icon
---
::
```